### PR TITLE
link https://www.freecodecamp.org added, case change in doc

### PR DESCRIPTION
**Pull request by lakshmipriyamukundan**
Its a doc change.

1.Case change for letter Women who Code to Women Who Code.
2.freecodecamp link added in one line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Our goal is to keep things simple and not reinvent wheels. So far we have only t
 2. Set up the API endpoints.
 3. Build the web client and let other developers use the API to build mobile clients and voice interface clients.
 
-Quincy Larson is the project lead. freeCodeCamp.org will start "dogfooding" this as soon as possible with several of its local study groups.
+Quincy Larson is the project lead. [FreeCodeCamp](https://www.freecodecamp.org) will start "dogfooding" this as soon as possible with several of its local study groups.
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
Quincy Larson is the project lead. [freeCodeCamp](https://www.freecodecamp.org) will start "dogfooding" this as soon as possible with several of its local study groups.
```

FCC is camel case.

**Comment by wuilliam321** [comment]
Don't forget to follow the contribution guide: https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md

**Comment by nik-john** [comment]
@all-contributors please add @lakshmipriyamukundan   for doc

**Comment by nik-john** [comment]
@all-contributors please add @lakshmipriyamukundan for doc

**Pull request merged**